eclass/qt5-build.eclass

```diff
@@ -631,7 +631,7 @@ qt5_base_configure() {
 		-no-compile-examples
 
 		# disable rpath on non-prefix (bugs 380415 and 417169)
-		$(usex prefix '' -no-rpath)
+		$(has prefix ${IUSE} && $(usex prefix '' -no-rpath))
```

Comment:
I'm confused, what does it mean if `prefix` is not in IUSE? in any case, the logic seems wrong, we should always pass `-no-rpath` unless we're on prefix.

Also, use `in_iuse` instead of `has`.

Reply:
ah yes, too fast there. `!use_if_iuse` seems useful.

Reply:
From what I understood, the fact that Portage does not throw an error if `prefix` (among several others, such as elibc_glibc, kernel_linux) is used without definition, is wrong according to PMS.